libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -310,13 +314,13 @@
     It returns the properly formatted SELECT query."
   [^String user ^String zone ^String parent-path ^String info-type-cond]
   (let [group-query "SELECT group_user_id FROM groups"]
-    (str "WITH groups    AS (" (mk-groups user zone) "),
-               objs      AS (" (mk-unique-objs-in-coll parent-path ) "),
-               file_avus AS (" (mk-obj-avus "SELECT data_id FROM objs") ")
-         " (mk-count-objs-of-type "objs" "file_avus" group-query info-type-cond))))
+    [[(mk-temp-table "groups" (mk-groups user zone))]
+     [(mk-temp-table "objs" (mk-unique-objs-in-coll parent-path))]
+     [(mk-temp-table "file_avus" (str "SELECT object_id, meta_attr_value, meta_attr_name FROM (" (mk-obj-avus "SELECT data_id FROM objs") ") obj_avus"))]
```

Comment:
this (and other ones below) have this subselect structure because otherwise they don't work because there's some conflicting column names. extra credit work for me or a reviewer would be to see if everywhere that uses `mk-obj-avus` can just be limited to these so we don't need it